test/parallel/test-child-process-spawn-typeerror.js

```diff
@@ -33,12 +33,11 @@ const invalidArgValueError =
   common.expectsError({ code: 'ERR_INVALID_ARG_VALUE', type: TypeError }, 14);
 
 const invalidArgTypeError =
-  common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 12);
+  common.expectsError({ code: 'ERR_INVALID_ARG_TYPE', type: TypeError }, 13);
 
 assert.throws(function() {
-  const child = spawn(invalidcmd, 'this is not an array');
-  child.on('error', common.mustNotCall());
```

Comment:
Any reason you removed this line? It tests that the `error` event is not emitted on the child process.

Reply:
Oh, upon reflection, I guess it's entirely unnecessary. If `spawn()` throws, then the next line is never executed, so it's not *really* testing anything. (If `spawn()` doesn't throw, then the test fails.)